### DISCUSSION TITLE
Installer unsigned ?

### DISCUSSION THREAD

**Discussion by JimLakis** [sqlitebrowser/sqlitebrowser#3398] [Q&A]
My system throws a fit when I try to install DB browser for Sqlite on my Windows 10 system. It claims that the software is **unsigned**. Why would this be the case?

**Comment by justinclift** [comment]
The official release download for v3.12.2 is signed.  It sounds like you're grabbing it from the wrong place maybe?

Though, maybe something else is going on.  Any chance there's software installed on your system that might be mucking around with things?

**Reply by JimLakis** [reply]
Other than the usual desktop maintenance and security efforts, what tools that I have don't seem to be indicating issues on my system leading to something "mocking around". But who knows.

**Comment by justinclift** [comment]
On, that's interesting.  Just checked my email, and our monitoring service had an alert from a few hours ago that our main download server (https://download.sqlitebrowser.org) had stopped serving downloads.

Looks like the process locked up somehow, without properly crashing.  That's pretty rare.

I've just restarted it, and it's working again now.  So, I'm thinking that you may have gotten a partial download or something, which would throw off signatures and similar.

Try again now, and just for good measure maybe see if the new download is different from your original one?

**Comment by JimLakis** [comment]
I downloaded the appropriate .zip file from the link you provided above into a user account, extracted it and launched the DB Browser for SQLite.exe file. The DB Browser for SQLite window interface appeared and no signature exceptions where thrown during the process, so for now I'll say that we resolved the main issue of this thread. Thank you for your help.

**Reply by justinclift** [reply]
Awesome. :smile:
